pkg/statistics/handle/globalstats/global_stats.go

```diff
@@ -151,7 +151,7 @@ func MergePartitionStats2GlobalStatsByTableID(
 // These values originally came from the analyzeOptionDefault structure in the planner/core/planbuilder.go file.
 var analyzeOptionDefault = map[ast.AnalyzeOptionType]uint64{
 	ast.AnalyzeOptNumBuckets: 256,
-	ast.AnalyzeOptNumTopN:    20,
```

Comment:
20 is a wrong value compared with the current non-partitioned table. But, yes, the previous merging operation is too slow. So I want to change it together with this pr.

Reply:
Undo it and change it later for smaller diff.